lib/ts/recipe/emailpassword/recipe.ts

```diff
@@ -71,6 +77,23 @@ export default class Recipe extends RecipeModule {
                 : new EmailVerificationRecipe(recipeId, appInfo, isInServerlessEnv, {
                       ...this.config.emailVerificationFeature,
                   });
+        {
+            let override = (
+                originalImplementation: EmailDelvieryRecipeInterface<TypeEmailPasswordEmailDeliveryInput>
+            ) => originalImplementation;
+            if (this.config.emailDelivery !== undefined && this.config.emailDelivery.override !== undefined) {
+                override = this.config.emailDelivery.override;
+            }
+            let builder = new OverrideableBuilder(
+                EmailDeliveryRecipeImplementation(this.emailVerificationRecipe, this.config.emailDelivery.service)
+            );
+            let emailDeliveryRecipeInterfaceImpl = builder.override(override).build();
+            this.emailDelivery = new EmailDeliveryRecipe(recipeId, appInfo, {
+                service: this.config.emailDelivery.service,
+                recipeImpl: emailDeliveryRecipeInterfaceImpl,
+            });
+        }
```

Comment:
This can be condensed to:
```js
this.emailDelivery = new EmailDeliveryRecipe(recipeId, appInfo, {
                defaultRecipeImpl: EmailDeliveryRecipeImplementation(this.emailVerificationRecipe, this.config.emailDelivery.service),
                override: this.config.emailDelivery.override,
            });
```
If you pass the default impl to `new EmailDeliveryRecipe` and let it take care of override

Reply:
Fixed in my PR